float32     f_secs
int32       runlevel
int32       sublevel
int32[2]    type
int32[6]    pos
int32[6]    pos_d
duration    dt
int32[16]   encVals
float32[16] tau
float32[16] mpos
float32[16] jpos
float32[16] mvel
float32[16] jvel
float32[16] mpos_d
float32[16] jpos_d
float32[16] encoffsets
